Controller/DbControllers/UserController.cs

```diff
@@ -79,12 +80,17 @@ public void MakeArtist(User user)
         {
             user.RoleID = 3;
             UpdateItem(user);
+            new ArtistController(new DatabaseContext()).CreateItem(new Artist() {ArtistName = user.Username}); // change user.Username to artist name
         }
 
         public void RevokeArtist(User user)
         {
             user.RoleID = 2;
             UpdateItem(user);
+
+            var artistID = new ArtistController(new DatabaseContext()).GetList()

```

Comment:
Kan dit niet gewoon in de ArtistController?
Ik denk dat dit vaker gebruikt moet worden, of zie ik dat verkeerd?

Reply:
klopt helemaal